web_src/js/index.js

```diff
@@ -2,7 +2,7 @@
 /* exported timeAddManual, toggleStopwatch, cancelStopwatch, initHeatmap */
 /* exported toggleDeadlineForm, setDeadline, updateDeadline, deleteDependencyModal, cancelCodeComment, onOAuthLoginClick */
 
-import 'jquery.are-you-sure';
+import 'jquery.are-you-sure/jquery.are-you-sure';
```

Comment:
This seems unnecessary as that file is already the `main` export:

```js
$ jq .main node_modules/jquery.are-you-sure/package.json
"jquery.are-you-sure.js"

Reply:
My vscode lint was complaining as to if you are importing directory it should have index.js if not full path to js file needs to be specified. But anyway this is out of scope for this PR

Reply:
Sounds like a bug in your linter probably cause by the dot in the module name which is very unusual but valid.